src/picasso-def/components/defines.js

```diff
@@ -2,6 +2,6 @@ const TREEMAP_DEFINES = Object.freeze({
   HEADER_HEIGHT: 24,
   COLLAPSED_HEADER_HEIGHT: 4,
   OUTER_PADDING: 2,
-  STROKE_WIDTH: 0.5,
+  STROKE_WIDTH: 1,
```

Comment:
maybe combine with changing OUTER_PADDING to 1 ?

Reply:
The outer padding 2 looks quite similar to old treemap